src/main/java/icbm/classic/api/missiles/cause/IMissileCause.java

```diff
@@ -22,7 +22,7 @@ public interface IMissileCause extends IBuildableObject {
      */
     default IMissileCause getRootCause() {
         final IMissileCause cause = getPreviousCause();
-        if(cause != null) { //TODO add logic to prevent infinite loop
+        if (cause != null) { //TODO add logic to prevent infinite loop
```

Comment:
Formatter is causing way too much code churn. Going to make the review process take long in the future.

Reply:
Keep in mind, this isn't a dead project looking for a new owner to take over and rework it. I'm still actively working on features for it in 1.12... only haven't had time to port it due to health and work.